test/lib/jdk/test/lib/hprof/parser/Reader.java

```diff
@@ -168,16 +172,18 @@ public static String getStack(String heapFile, int debugLevel)
                             = new HprofReader(deCompressedFile, in2, dumpNumber,
                                               true, debugLevel);
                         r.read();
+                        out.delete();
```

Comment:
I thought this was going to handled by the separate RFE to fix the general logic? And you don't delete before throwing below. There should be a finally clause to handle deletion consistently.